bin/templates/default.tmpl

```diff
@@ -1,6 +1,6 @@
 import angular from 'angular';
 
-var <%= componentName %> = angular.module('<%= componentName %>', []);
```

Comment:
Are dashes allowed in Angular module names? A quick google didn't turn up anything interesting. Anyway, I've only seen camel case I think.